charts/prefect-orion/README.md

```diff
@@ -32,6 +32,7 @@ Prefect orion application bundle
 | api.autoscaling.maxReplicas | int | `100` |  |
 | api.autoscaling.minReplicas | int | `1` | Minimum autoscaling replica count |
 | api.autoscaling.targetCPUUtilizationPercentage | int | `80` |  |
+| api.commonLabels | object | `{}` |  |
```

Comment:
seems reasonable. we might consider naming this extraLabels in case we want any fixed/unremovable Chart-controlled labels